koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -628,7 +628,7 @@ SELECT azure.uuid as azure_uuid,
     max(azure.subscription_name) as subscription_name,
     max(nullif(azure.resource_location, '')) as resource_location,
     max(azure.unit_of_measure) as unit_of_measure,
-    max(cast(azure.usage_quantity as decimal(24,9))) as usage_quantity,
+    max(cast(azure.quantity as decimal(24,9))) as quantity,
```

Comment:
these shouldnt change since they are being pulled from https://github.com/project-koku/koku/pull/5220/files#diff-67da6cc06454258dfe044f57b1202cf5b77f6062930c18c38eb05f934017e060R14 

In short we should only be updating `usagequantity` not `usage_quantity` as thats our normalised column name.